tests/backup/backup_helper.go

```diff
@@ -4427,3 +4430,86 @@ func UpdateCluster(clusterName string, clusterUid string, kubeConfigPath string,
 	}
 	return status, err
 }
+
+// CreateRole creates role with given services and apis in px-backup datastore and also add role to keycloak.
+func CreateRole(roleName backup.PxBackupRole, svcs, apis []string, ctx context.Context) error {
+	err := backup.AddRoleToKeycloak(roleName, "custom-role")
+	if err != nil {
+		return err
+	}
```

Comment:
Do we really need to do this?
When you try to create a role from UI we are not providing the role ID. We are just passing the role name along with APIs and Services right?

Reply:
Yes this is the workflow, If directly made call to px-backup, keycloak will be unware of the newly added role and we wont be able to add role later to user. Actual UI flow remains same, when we create role in UI, first entry is made in keycloak and request is given to px-backup.